acceptance/bin/callserver.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+import sys
+import os
+import json
+import subprocess
+from urllib.parse import urlencode
+
+env = {}
+for key, value in os.environ.items():
+    if len(value) > 10_000:
+        sys.stderr.write(f"Dropping key={key} value len={len(value)}\n")
+        continue
```

Comment:
Why is this needed?

Reply:
Fail explicitly in case of environment is weird rather than experience slowdown.